l1interop/l1sseclient.go

```diff
@@ -227,16 +226,6 @@ func (l *l1SseClient) Start(nConnectedl1s *atomic.Uint64) error {
 
 		n = nConnectedl1s.Dec()
 		log.Infow("lost connection to L1", "l1", l.l1Addr, "nL1sConnected", n)
-		reportConnectionStatus(n)
-	}
-}
-
-func reportConnectionStatus(nConnectedl1s uint64) {
```

Comment:
@bajtos Removed this as this is now subsumed by the `logL1Connectivity` function in main.go

Reply:
Makes sense 👍🏻 

The new version based on channels is much easier to reason about 👏🏻